docs/configuration.md

```diff
@@ -807,6 +814,14 @@ Apart from these, the following properties are also available, and may be useful
     Allows jobs and stages to be killed from the web UI.
   </td>
 </tr>
+<tr>
+  <td><code>spark.ui.liveUpdate.period</code></td>
+  <td>100ms</td>
+  <td>
+    How often to update live entities. For live applications, this avoids a few
```

Comment:
Could you add `-1 means "never update" when replaying applications, meaning only the last write will happen.`  before `For live applications`?

Reply:
Thanks updated.